.github/workflows/build-docs.yaml

```diff
@@ -30,32 +30,3 @@ jobs:
         run: npm run build
         env:
           NODE_OPTIONS: --max_old_space_size=4096
-      - name: Remove https redirect
-        run: |
-          sed -i '/SERVER_PORT/d;/SERVER_NAME/d' build/.htaccess
-      - name: Run web server
-        run: |
-          docker run -d --name webserver -v "$PWD/build":/app -p 8888:8080 bitnami/apache:2.4.54
-          echo "container_ip=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' webserver)" >> $GITHUB_ENV
-      - name: Download sitemap from production
-        run: curl -sL https://docs.camunda.io/sitemap.xml | grep -oP '<loc>\K.*?(?=</loc>)' | sed "s!https://docs.camunda.io!http://${{ env.container_ip }}:8080!g" > sitemap.prod.txt
-      - name: Prepare product links
-        run: grep 'https://docs.camunda.io' product-links.txt | sed "s!https://docs.camunda.io!http://${{ env.container_ip }}:8080!g" > product-links.prod.txt
-      - name: Waiting for URL available
-        uses: nev7n/wait_for_response@v1
-        with:
-          url: "http://${{ env.container_ip }}:8080/"
-      - name: Validate sitemap links
-        uses: lycheeverse/lychee-action@v1.6.1
-        with:
-          args: "sitemap.prod.txt"
-          fail: true
-      - name: Validate product links
-        uses: lycheeverse/lychee-action@v1.6.1
-        with:
-          args: "product-links.prod.txt"
-          fail: true
-      - name: Check internal links
-        uses: untitaker/hyperlink@0.1.27
-        with:
-          args: build/
```

Comment:
This is potentially controversial. The sitemap & product link checks are definitely not something we want to run, because this is an isolated site containing only one old version. But the internal links of the app is probably still worth checking, if it's not much work.

Unfortunately it isn't "not much work"! I'd have to alter the steps above that run the webserver to run the app in a /0.25 folder, or else [the internal link-checking would fail](https://github.com/camunda/camunda-platform-docs/actions/runs/5159889016/jobs/9295356005) because of the baseURL of /0.25.

My feeling is that it's fine to remove this link-checking. It's an unsupported version of the docs, and if an internal link gets broken over time, I think we just say forget it.

But if you feel strongly that I should get this internal link-checking working, let me know!!!